src/types/value.rs

```diff
@@ -87,11 +87,13 @@ impl<T: Into<Bytes>> From<T> for Value {
     }
 }
 
-impl<T: Into<Bytes>> From<Option<T>> for Value {
-    fn from(b: Option<T>) -> Value {
-        match b {
-            Some(b) => Value::new_normal(b.into()),
-            None => Value::new_null(),
+#[cfg(cdrs_orm)]
+impl <T: serde::Serialize + Into<Bytes>> Into<Bytes> for Option<T> {
```

Comment:
This is needed for the orm

Reply:
This breaks existing API by removing functionality.

Reply:
I will check if I can make the orm work without this impl

Reply:
I can not get the ORM to work without the serde dependency and this new `impl`. I doubt anyone uses the current impl though. 

For the time being, I close this PR because I don't have the time to create a workaround for this impl. Maybe in the future I will have another look.

Reply:
Maybe try using newtypes in the ORM, instead of serializing cdrs structures directly?

Reply:
Than I am still stuck with the issue that I can not call `serde_json::to_string(&self).unwrap().into()`, regardless if I introduce newtypes or not (source can be found here: https://github.com/Jasperav/cdrs_orm/blob/13e0a377b0fac6aa80f91b393ee1e44723afe4cc/cdrs_db_mirror/src/db_json.rs#L13).

It is a bit of a dependency issue, cdrs_orm relies on cdrs but cdrs must also do some work for the orm (by providing an impl or do stuff based on a field is annoted in the derive-helper crate). That is a bit odd indeed, but it takes a lot of time to refactor that and I have no clue how... maybe in the future I will have another look if the repo's can be merged if you want to remain backward compatible for the impl functionality.

Reply:
You should not work on cdrs types directly - this leads to dependency hell. Try to wrap things into your own types and impl things on them instead.